cms/envs/production.py

```diff
@@ -111,7 +111,7 @@ def get_env_setting(setting):
 BROKER_CONNECTION_TIMEOUT = 1
 
 # For the Result Store, use the django cache named 'celery'
```

Comment:
@navinkarkera Hey! Can you also update the comment above to reflect that the result backend is now customizable, but falls back to `django-cache` if nothing else is configured?

Reply:
@giovannicimolin Thank you! Updated the comment.